src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/VirtualDocumentFactoryBase.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
```

Comment:
```suggestion
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

﻿using System;
```

Reply:
Added, thanks!

---
In reply to: [451969326](https://github.com/dotnet/aspnetcore-tooling/pull/2201#discussion_r451969326) [](ancestors = 451969326)